angel_system/impls/detect_activities/pytorchvideo_slow_fast_r50.py

```diff
@@ -139,12 +147,20 @@ def detect_activities(
         # Get the predicted classes
         post_act = torch.nn.Softmax(dim=1)
         preds = post_act(preds)
-        pred_classes = preds.topk(k=5).indices
+        top_preds = preds.topk(k=5)
 
         # Map the predicted classes to the label names
+        pred_classes = top_preds.indices
```

Comment:
I added some comments to hopefully make it a bit more clear. The output of the model is a (1,400) tensor and that topk result is a (1,k) tensor. So that's why I am taking just the first index.